helm-chart/templates/configmap.yaml

```
@@ -7,27 +7,26 @@ metadata:
     {{ include "json-rpc-relay.labels" . | nindent 4 }}
 data:
 {{- if and .Values.config.local .Values.config.local.HEDERA_NETWORK }}
-  HEDERA_NETWORK: {{ .Values.config.local.HEDERA_NETWORK | toJson }}
+  HEDERA_NETWORK: {{ .Values.config.local.HEDERA_NETWORK | toJson | quote}}
 {{- else if .Values.config.hosted.HEDERA_NETWORK }}
-  HEDERA_NETWORK: {{ .Values.config.hosted.HEDERA_NETWORK }}
+  HEDERA_NETWORK: {{ .Values.config.hosted.HEDERA_NETWORK | quote }}
 {{- else }}
   HEDERA_NETWORK: {{ required "A valid HEDERA_NETWORK must be present in either .Values.config.local.HEDERA_NETWORK or .Values.config.hosted.HEDERA_NETWORK " (or .Values.config.local.HEDERA_NETWORK .Values.config.hosted.HEDERA_NETWORK) }}
 {{- end }}
-  CHAIN_ID: {{ .valu}}
-  MIRROR_NODE_URL: {{ .Values.config.MIRROR_NODE_URL  }}
-  LOCAL_NODE: {{ .Values.config.LOCAL_NODE }}
-  SERVER_PORT: {{ .Values.config.SERVER_PORT }}
-  CHAIN_ID: {{ .Values.config.CHAIN_ID }}
-  DEFAULT_RATE_LIMIT: {{ .Values.config.DEFAULT_RATE_LIMIT }}
-  TIER_1_RATE_LIMIT: {{ .Values.config.TIER_1_RATE_LIMIT }}
-  TIER_2_RATE_LIMIT: {{ .Values.config.TIER_2_RATE_LIMIT }}
-  TIER_3_RATE_LIMIT: {{ .Values.config.TIER_3_RATE_LIMIT }}
-  LIMIT_DURATION: {{ .Values.config.LIMIT_DURATION }}
-  HBAR_RATE_LIMIT_TINYBAR: {{ .Values.config.HBAR_RATE_LIMIT_TINYBAR }}
-  HBAR_RATE_LIMIT_DURATION: {{ .Values.config.HBAR_RATE_LIMIT_DURATION }}
-  ETH_GET_LOGS_BLOCK_RANGE_LIMIT: {{ .Values.config.ETH_GET_LOGS_BLOCK_RANGE_LIMIT }}
-  RATE_LIMIT_DISABLED: {{ .Values.config.RATE_LIMIT_DISABLED }}
-  DEV_MODE: {{ .Values.config.DEV_MODE }}
-  MIRROR_NODE_RETRIES: {{ .Values.config.MIRROR_NODE_RETRIES }}
-  MIRROR_NODE_RETRY_DELAY: {{ .Values.config.MIRROR_NODE_RETRY_DELAY }}
-  MIRROR_NODE_LIMIT_PARAM: {{ .Values.config.MIRROR_NODE_LIMIT_PARAM }}
+  MIRROR_NODE_URL: {{ .Values.config.MIRROR_NODE_URL | quote }}
+  LOCAL_NODE: {{ .Values.config.LOCAL_NODE | quote }}
+  SERVER_PORT: {{ .Values.config.SERVER_PORT | quote }}
+  CHAIN_ID: {{ .Values.config.CHAIN_ID | quote }}
+  DEFAULT_RATE_LIMIT: {{ .Values.config.DEFAULT_RATE_LIMIT | quote }}
+  TIER_1_RATE_LIMIT: {{ .Values.config.TIER_1_RATE_LIMIT | quote }}
+  TIER_2_RATE_LIMIT: {{ .Values.config.TIER_2_RATE_LIMIT | quote }}
+  TIER_3_RATE_LIMIT: {{ .Values.config.TIER_3_RATE_LIMIT | quote }}
+  LIMIT_DURATION: {{ .Values.config.LIMIT_DURATION | quote }}
+  HBAR_RATE_LIMIT_TINYBAR: {{ .Values.config.HBAR_RATE_LIMIT_TINYBAR | quote }}
+  HBAR_RATE_LIMIT_DURATION: {{ .Values.config.HBAR_RATE_LIMIT_DURATION | quote }}
+  ETH_GET_LOGS_BLOCK_RANGE_LIMIT: {{ .Values.config.ETH_GET_LOGS_BLOCK_RANGE_LIMIT | quote }}
+  RATE_LIMIT_DISABLED: {{ .Values.config.RATE_LIMIT_DISABLED | quote }}
+  DEV_MODE: {{ .Values.config.DEV_MODE | quote }}
+  MIRROR_NODE_RETRIES: {{ .Values.config.MIRROR_NODE_RETRIES | quote }}
+  MIRROR_NODE_RETRY_DELAY: {{ .Values.config.MIRROR_NODE_RETRY_DELAY | quote }}
+  MIRROR_NODE_LIMIT_PARAM: {{ .Values.config.MIRROR_NODE_LIMIT_PARAM | quote }}
```

Can you add `CLIENT_TRANSPORT_SECURITY` seems it was missed when added  in previous PRs
```suggestion
  MIRROR_NODE_LIMIT_PARAM: {{ .Values.config.MIRROR_NODE_LIMIT_PARAM | quote }}
  CLIENT_TRANSPORT_SECURITY: {{ .Values.config.CLIENT_TRANSPORT_SECURITY | quote }}
```